runtime/support.py

```diff
@@ -62,6 +62,7 @@
 import zipfile
 import zipimport
 
+PAR_DIRECTORY = os.path.join(os.path.expanduser("~"), '.pex', 'par')
```

Comment:
Does `~/.subpar` work?

Reply:
Keeping it consistent with how pex works - it defaults to ~/.pex for writing. Since devs are already familiar with it, don't want to add an additional layer of learning a different pex/par cache directory. We can revisit this in the future.

Reply:
Would be nice to have some env var to override this path, just in case. If we make any changes in the future can we include that as well